rtb/verbali/interni/VI_2024-11-10_ver1.0.0.typ

```diff
@@ -0,0 +1,77 @@
+- Il redattore del verbale dovrà popolare la tabella delle decisioni collegando i codici delle attività con le rispettive issue.
+- I nomi delle issue devono essere ben descrittivi perché saranno inseriti nella Tabella delle decisioni.
+- I codici delle attività sono strutturati nel seguente modo: *[X][\#ID issue]* dove X è una lettera che identifica univocamente la repository, seguita dall'ID della issue scritta con quattro cifre così da mantenere una lunghezza fissa fino alla issue numero 9999 per ciascuna repository. Le lettere che identificano le repository al momento sono *D* per la documentazione e *S* per il sito. Nel caso le attività non corrispondessero a delle issue perché di natura organizzativa (ad esempio richiedere un incontro all'azienda proponente) si aggiunge il codice *ORG*.
+
+== Aggiornamento GitHub Action
+Per correttezza nei confronti dell'azienda proponente che deve firmare i verbali esterni abbiamo deciso di non pubblicare questi ultimi fino a presa visione della controparte. Inoltre un passaggio della verifica dei documenti è la compilazione dei file Typst. Abbiamo pianificato quindi i seguenti cambiamenti per la GitHub Action:
+- Deve compilare i file .typ modificati in modo da rilevare errori sintatici e rendere il documento di facile lettura per i verificatori. Inoltre compilando solo i file modificati evitiamo il sovraccarico con l'aumentare della documentazione.
+- Deve pubblicare nel sito i documenti compilati ad esclusione dei verbali esterni. Questi, una volta firmati dalla controparte, vengono aggiunti manualmente nella cartella dei verbali esterni in formato .pdf e pubblicati automaticamente.
+
+== Diario di Bordo
+Abbiamo formato i punti da discutere per la lezione dedicata ai "Diari di Bordo" di martedì 12 novembre. Inoltre ci siamo accordati sul fatto di dedicare 10 minuti alla fine dell'ultima riunione di ciascuna settimana per delineare il diario di bordo della settimana successiva. In caso nascessero problematiche improvvise ci impegnamo a riunirci brevemente sulla piattaforma Discord il fine settimana.
+
+== Glossario e README
+Per evitare malinterpretazioni tra i termini abbiamo aggiunto un glossario da popolare col proseguimento del progetto. Abbiamo inoltre aggiunto le informazioni generali alle repository nel file standard README.md.
+
+= Conclusioni
+Come molte riunioni nel primo periodo del progetto questa è servita a modellare nuovamente il Way of Working del gruppo oltre a incrementare la struttura dell'organizzazione. Ci impegnamo quindi a rispettare i punti definiti precedentemente in modalità asincrona e a incontrarci il prima possibile con l'azienda proponente. In caso non fosse possibile pianificheremo una riunione interna.
+
+= Tabella delle decisioni
+#backlog(
+  content: (
+    "ORG", "Andrea Precoma", "Invito di un incontro all'azienda proponente via e-mail",
```

Comment:
Forse invece di "Invito di un incontro" scriverei  "Invito a un incontro" oppure "Richiesta di incontro". 
Ma è veramente una piccolezza, il resto mi sembra vada più che bene. Mi piace la scelta di usare "ORG", rende l'idea.

Reply:
Concordo. Non è necessario, ma va bene. @ErPreco poi gli identificativi secondo me dovremmo metterli sulle norme di progetto

Reply:
Richiesta di incontro effettivamente suona meglio. Per gli ID chiaramente andranno inseriti nelle norme di progetto ma in mancanza di quelle per ora le ho inserite qua, come molte decisioni prese nel verbale scorso.